### PR TITLE
[MO] - [system test] -> persistent storage size

### PR DESCRIPTION
**Pull request by Seequick1**
### Type of change

- Enhancement / new feature

### Description

System test, that is checking if storage size was set correctly.

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [x] Write tests
- [ ] Make sure all tests pass

### CODE REVIEW

**Comment by scholzj** [comment]
@strimzi-ci run tests testcase=KafkaST

File: systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java [hunk 1]
```diff
@@ -1407,6 +1408,46 @@ void verifyVolumeNamesAndLabels(int kafkaReplicas, int diskCountPerReplica, int
         }
     }
 
+    @Test
+    void testPersistentStorageSize() throws Exception {
+        String diskSizeGi = "70Gi";
```
Comment (ppatierno) [inline systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java]: why not using a couple of different storage sizes just to test that actually they will be set differently?

Comment (Seequick1) [reply systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java]: Yes, that's a good point but question is. Is there any special storage size that's has to be tested? I can do this test @parametrized.

Comment (ppatierno) [reply systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java]: I don't think so. My suggestion was more about testing that if you ask for 5Gi on volume 0 and 10Gi on volume 1 it doesn't turn in the other was around (10Gi on volume 0 and 5Gi on volume 1) so checking that operator is doing it in the right way. Using one size, it's something you cannot test.

**Pull request merged**